crates/sui-framework/packages/move-stdlib/sources/u256.move

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+#[defines_primitive(u256)]
+module std::u256 {
+    /// Return the larger of `x` and `y`
+    public fun max(x: u256, y: u256): u256 {
+        std::macros::num_max!(x, y)
+    }
+
+    /// Return the smaller of `x` and `y`
+    public fun min(x: u256, y: u256): u256 {
+        std::macros::num_min!(x, y)
+    }
+
+    /// Return the absolute value of x - y
+    public fun diff(x: u256, y: u256): u256 {
+        std::macros::num_diff!(x, y)
+    }
+
+    /// Calculate x / y, but round up the result.
+    public fun divide_and_round_up(x: u256, y: u256): u256 {
+        std::macros::num_divide_and_round_up!(x, y)
+    }
+
+    /// Return the value of a base raised to a power
```

Comment:
Nit: replicate comment like there are for the other `pow` functions in other integer modules

Reply:
Sorry, what comment is missing?